### PR TITLE
#25 Article: Meet with users often

### PR DESCRIPTION
**Pull request by bruderol**
Article by @mdf88

### CODE REVIEW

File: articles/MeetUsers [hunk 1]
```diff
@@ -0,0 +1,214 @@
+# meet with users - often
```
Comment (bruderol) [inline articles/MeetUsers]: Titles should be capitalized: Meet With Users - Often
(as usual in English)
see also http://www.businesswritingblog.com/business_writing/2010/02/words-to-capitalize-in-titles-and-headings.html - maybe we have to define which exact capitalization rules we want to enforce in the book

**Comment by bruderol** [comment]
The article file has to be renamed to have `.MD` as an ending!

File: articles/MeetUsers [hunk 2]
```diff
@@ -0,0 +1,214 @@
...
+_Meeting with users, the more the better, is essential to create great products._
+
+Even though I - as a UX professional - must stress the importance of listenting to the users, I still have to admit that the above statement is inaccurate and misleading. With just meeting users nothing is achieved.
+
+Having started with saying that the title of the article I'm writing is, well, rubbish, I need to put up a real big headline to correct this:
+
+## We need to experience what users will do and what they will need and will expect from our products when creating them.
```
Comment (bruderol) [inline articles/MeetUsers]: etwas gar lang für einen Abschnitt-Titel

File: articles/MeetUsers [hunk 3]
```diff
@@ -0,0 +1,214 @@
...
+* Not every function has the same importance or difficulty. Test and optimize only what really needs to be optimized.
+* Don't let users be the first to test. Take a real life story and play it through. You will find many quirks in your product.
+
+### "Understand": get inspired by life
+
+Ok, I did put that point up already. But don't forget: once your product is up and running and being bought and used, collect and evaluate real feedback. How did life change? What new needs did we create? What opportunities are there for us? So go back to the users and try again to figure out, whether your product needs optimizsation, enhancenemnts, new features or even replacing with yet another product. 
+
+_ It's about inspiration to create the next even greater product while you're competition tries to copy your current product_
+
+
```
Comment (bruderol) [inline articles/MeetUsers]: Vermute dass der Artikel etwas gar lang ist um auf zwei Seiten zu passen.

File: articles/MeetUsers (hunk 1 above)
Comment (mdf88) [inline articles/MeetUsers]: Yes, some style guides would be a good idea

File: articles/MeetUsers (hunk 3 above)
Comment (mdf88) [inline articles/MeetUsers]: Auf jeden Fall zu lang. Die Frage ist: was kann man streichen? Ich bitte um Vorschläge: was findet ihr wertvoll? Was überflüssig.

Comment (bruderol) [inline articles/MeetUsers]: ja, gute Idee - erst mal alles sammeln, dann verdichten ;-) 👍

**Comment by abeggchr** [comment]
close pull request, mfd88 will create a new one when ready

**Comment by bruderol** [comment]
@mdf88 @abeggchr I rather recommend to open Pull Request early - such that everybody can give feedback early. Later changes will be reflected in the PR automatically.

**Pull request closed without merging**